src/arrayMethodReduce.js

```diff
@@ -5,7 +5,19 @@
  */
 function applyCustomReduce() {
   [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+    let accumulator = initialValue;
+    let start = 0;
+
+    if (arguments.length <= 1) {
```

Comment:
Can use more obvious condition.
```suggestion
    if (arguments.length < 2) {
```